ladybug_comfort/_extend_ladybug.py

```diff
@@ -0,0 +1,130 @@
+# coding=utf-8
+"""Collection of methods that extends current objects in ladybug core library.
+
+This module is imported in __init__.py and applies the changes on runtime.
+"""
+from __future__ import division
+
+from ladybug.epw import EPW
+
+from .collection.utci import UTCI
+from .collection.pmv import PMV
+from .collection.solarcal import OutdoorSolarCal
+
+
+def get_universal_thermal_climate_index(self, include_wind=False, include_sun=False,
```

Comment:
Can you explain why you feel that way?

Most weather service websites like [weather underground](https://www.wunderground.com/) only use the air temperature, humidity, and wind speed to get the "feels like" temperature, essentially excluding the effect of sun.  This seems to be fairly common practice among most TV meteorologists as well.  I might have set the default to do the same but I thought it was better to be consistent about including or not including everything.

It might also be worth noting that including both sun and wind can really increase the calculation time, especially for standard effective temperature.  The run time goes up by a factor of 10 just because the conditions are further from steady state.

Reply:
Because it is closer to the reality. At least the way we use it in the workshops when you remove sun or wind the pattern doesn't correlate much with the real experience for people who leave in that climate. Seeing that most of people use the default as is, I think our defaults should be set to reflect what is closest to the use of our users. Again this is based on my assumption of how the users would use UTCI for decision making.

I won't be concerned about the run time. I think it comes down to which option is the better choice for most of our users and if takes longer then let it be.

Reply:
Ok. You convinced me.  You are right to point out that people in our workshops are often surprised by results of just dry bulb and relative humidity.  I will change the default to include both wind and sun.